critiquebrainz/data/dump_manager.py

```diff
@@ -93,26 +92,26 @@ def json(location, rotate=False):
     current_app.json_encoder = DumpJSONEncoder
 
     print("Creating new archives...")
-    for license in model.License.query.all():
-        safe_name = slugify(license.id)
+    for license in db.license.list_licenses():
+        safe_name = slugify(license["id"])
         with tarfile.open(os.path.join(location, "critiquebrainz-%s-%s-json.tar.bz2" %
                 (datetime.today().strftime('%Y%m%d'), safe_name)), "w:bz2") as tar:
             temp_dir = tempfile.mkdtemp()
             license_dir = os.path.join(temp_dir, safe_name)
             create_path(license_dir)
 
             # Finding release groups that have reviews with current license
```

Comment:
Might be a good idea to update that comment since you are here. It's not just "release groups" anymore.